airflow/operators/bash_operator.py

```diff
@@ -87,14 +88,14 @@ def execute(self, context):
 
                 self.sp = sp
 
-                logging.info("Output:")
-                line = ''
+                output = StringIO()
                 for line in iter(sp.stdout.readline, b''):
-                    line = line.decode(self.output_encoding).strip()
-                    logging.info(line)
+                    output.write(line)
+                output = output.getvalue()
```

Comment:
well i feel it exposes the output of the command you run to everyone. it may not be the case for everyone though.